src/main/resources/templates/passwordreset.html

```diff
@@ -23,7 +27,25 @@ <h2 class="team-heading team-heading-current-team" th:text="#{passwordReset.head
 		              </fieldset>
 		          </form>
 		    </div> <!-- row -->
+			<div class="cc-wraper">
+				<div class="cc-contactpop fixed">
+					<h1>Do you need Help? <span>Get in touch.</span></h1>
+					<div class="renseignement">
+						<i class="fa fa-times"></i>
+						<p>We will get back to you as soon as possible. </p>
+					</div>
+					<form>
+						<ul>
+							<li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
+							<li><i class=""></i><span>Email:</span><a href="mailto:support@ncl.sg"> support@ncl.sg</a></li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)